javascript/javascript2/week1/animalName.js

```diff
@@ -0,0 +1,31 @@
+const animalNames = [
+    'Contemplative Teal Beetle',
+    'Unwavering White Ridley',
+    'Sensible Fuchsia Mockingbird',
+    'Steady Plum Boa',
+    'Convivial Purple Cottontail',
+    'Bald Forest Deer',
+    'Silky Lake Owl',
+    'Fat Sea Snake',
+    'Scaled Cave Zebra',
+    'Juicy Glue Gun Kitty'
+];
+
+const spiritAnimal = document.querySelector('.spiritAnimal');
+const button = document.querySelector(".submit");
+button.innerHTML = "submit";
```

Comment:
This seems like something I'd set directly in the HTML.

Reply:
I thought we should do them in JS

Reply:
It's probably good training!